src/notation/qml/MuseScore/NotationScene/ZoomControl.qml

```diff
@@ -49,6 +49,8 @@ RowLayout {
         icon: IconCode.ZOOM_IN
         iconFont: ui.theme.toolbarIconsFont
 
+        width: 28
+        height: 28
```

Comment:
Better to use bindings in such situations:

width: 28
height: width

Reply:
Done!